ChangeLog.md

```diff
@@ -20,6 +20,8 @@ See docs/process.md for more on how version tagging works.
 
 4.0.13 (in development)
 -----------------------
+- The `MINIMAL_RUNTIME_STREAMING_WASM_COMPILATION` setting is only compatible
```

Comment:
Do we need an entry here? This was always the case, and this PR only adds a clear error. This entry makes it seem like the PR removes functionality.

lgtm without this (unless I'm missing something)

Reply:
I updated to make it more clear